server/llm/openai_assistant.py

```diff
@@ -16,9 +16,12 @@ class OpenAIAssistant(Assistant):
     # For now, just store context in memory.
     _context: list[ChatCompletionMessageParam] = None
     _default_prompt = ChatCompletionSystemMessageParam(
-        content="AI adopts role of meeting recorder to provide concise meeting summaries incl. discussed key items, "
-                "decisions and action items. Based on the meeting transcript, write a summary that helps document all "
-                "key aspects of the conversation.", role="system")
+        content="AI adopts role of meeting recorder to provide short meeting summaries including discussed key "
+                "items, decisions and action items. Based on the meeting transcript only, write a summary that helps "
+                "document all key aspects of the conversation. Structure the answer into digestible chunks. Do not "
+                "assume things that are not in the transcript. Answer without square brackets, tags, or timestamps."
+                "The summary should be no more than 6 sentences long.",
+                role="system")
```

Comment:
I asked ChatGPT (4) to review this prompt and here's what it recommended:

```
Based on the provided meeting transcript, please create a concise summary. Your summary should include:

1. Key discussion points.
2. Decisions made.
3. Action items assigned.

Keep the summary within six sentences, ensuring it captures the essence of the conversation. Structure it in clear, digestible parts for easy understanding. Rely solely on information from the transcript; do not infer or add information not explicitly mentioned. Exclude any square brackets, tags, or timestamps from the summary.
```

For comparison: this is the prompt it suggested for the cleaned up transcript:

```
Using the exact transcript provided in the previous messages, convert it into a cleaned-up, paragraphed format. It is crucial that you strictly adhere to the content of the provided transcript without adding or modifying any of the original dialogue. Your tasks are to:

1. Correct punctuation and spelling mistakes.
2. Merge broken sentences into complete ones.
3. Remove timestamps and transcript types.
4. Clearly indicate the speaker's name at the beginning of their dialogue.

Do not add any new content or dialogue that was not present in the original transcript. The focus is on cleaning and reformatting the existing content for clarity and readability.
```

I'm using the suggested prompt for the transcript in https://github.com/daily-demos/ai-meeting-assistant/blob/main/client/utils/api.js#L21-L30, but I'll leave it to you to change it here ☺️

Reply:
Updated to your (or chatgpt's :D) suggestion!